scripts/run_with_beaker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SOURCES_ARG="--source squad.latest:/squad --source glove.latest:/glove"
+RESULT_ARG="--result-path /output"
+GPU_ARG="--gpu-count=1"
+DETACH_ARG="--detach"  # or ""
+
+ECR_REPOSITORY=896129387501.dkr.ecr.us-west-2.amazonaws.com
+
+PARAM_FILE=$1
+EXPERIMENT_NAME=$2
+
+COMMIT=$(git rev-parse HEAD)
+IMAGE=$ECR_REPOSITORY/allennlp/allennlp-gpu:$COMMIT
+
+if [ ! -n "$PARAM_FILE" ] ; then
+  echo "USAGE: ./run_on_aws.sh PARAM_FILE [EXPERIMENT_NAME]"
+  exit 1
+fi
+
+if [ -n "$EXPERIMENT_NAME" ] ; then
+  EXPERIMENT_NAME_ARG="--name=$EXPERIMENT_NAME"
+else
+  EXPERIMENT_NAME_ARG=""
+fi
+
+
+set -e
+
+# Get temporary ecr login. For this command to work, you need the python awscli
+# package with a version more recent than 1.11.91.
+eval $(aws --region=us-west-2 ecr get-login --no-include-email)
+
+docker build -f Dockerfile.gpu.train -t $IMAGE . --build-arg PARAM_FILE=$PARAM_FILE
```

Comment:
You already bundle `allennlp/` into the docker image.  We could either commit the experiment json or keep it in a `.gitigore`ed file and bundle it in the docker image.  That way it doesn't need to be passed as a separate file, although it would need to be an argument to the command.

This feel a bit hacky, but it would avoid the need for an additional Docker image.